src/Bridge.ts

```diff
@@ -679,10 +688,29 @@ export class Bridge {
         await retry(() => this.as.botIntent.joinRoom(roomId), 5);
         if (event.content.is_direct) {
             const room = await this.setupAdminRoom(roomId, {admin_user: event.sender}, NotifFilter.getDefaultContent());
-            await this.as.botIntent.underlyingClient.setRoomAccountData(
+            await this.as.botClient.setRoomAccountData(
                 BRIDGE_ROOM_TYPE, roomId, room.accountData,
             );
         }
+
+        if (this.connectionManager?.isRoomConnected(roomId)) {
+            // Room has connections, don't setup a wizard.
+            return;
+        }
+
+        try {
+            // Otherwise it's a new room
+            if (this.config.widgets?.roomSetupWidget?.addOnInvite) {
+                if (await this.as.botClient.userHasPowerLevelFor(this.as.botUserId, roomId, "im.vector.modular.widgets", true) === false) {
+                    await this.as.botIntent.sendText(roomId, "Hello! To setup new integrations in this room, please promote me to a Moderator/Admin");
```

Comment:
It's mostly to avoid confusing users. We didn't want to say "Give me PL50" which is probably going to confuse users on clients like Element which don't expose that easily. Granted, this is a difficult line to tread. Maybe we need a link to documentation for things like this.

Reply:
Fair enough – this is the user-friendly option and it was instantly understandable to me when seeing it originally in your video. +1 to that.